src/wallet/rpc/spend.cpp

```diff
@@ -486,6 +486,27 @@ static std::vector<RPCArg> FundTxDoc(bool solving_data = true)
     return args;
 }
 
+static inline OutputType parseOutputType(const std::string& outputTypeStr)
+{
+    const auto outputType = ParseOutputType(outputTypeStr);
+    if (!outputType) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, strprintf("Unknown input type '%s'", outputTypeStr));
+    }
+
+    return *outputType;
+}
+
+static inline std::vector<std::string> parseStringList(std::stringstream& ss)
+{
+    std::vector<std::string> result;
+    std::string item;
+    while (ss.good()) {
+        getline(ss, item, ',');
+        result.push_back(item);
+    }
+    return result;
+}
```

Comment:
this function should also handle lists like `bech32, bech32m`, as most users will probably pass them with spaces. i tested that this currently fails by passing `bech32, bech32m` in the functional test.

Reply:
Thanks, I fixed it